cmd/prometheus/main.go

```diff
@@ -97,7 +97,7 @@ func Main() int {
 	reloadables = append(reloadables, remoteStorage)
 
 	var (
-		notifier       = notifier.New(&cfg.notifier)
+		notifier       = notifier.New(&cfg.notifier, prometheus.DefaultRegisterer)
```

Comment:
I think this should also go into `cfg.notifier` instead of being a separate argument. And then just preinitialize that field in the config's `init()` to the default registry: https://github.com/prometheus/prometheus/blob/master/cmd/prometheus/config.go#L85
```